configs/recognition/c3d/c3d_sports1m_16x1x1_30e_ucf101_rgb.py

```diff
@@ -0,0 +1,135 @@
+# model settings
+model = dict(
+    type='Recognizer3D',
+    backbone=dict(
+        type='C3D',
+        pretrained=  # noqa: E251
+        'https://download.openmmlab.com/mmaction/recognition/c3d/c3d_sports1m_pretrain_20201016-dcc47ddc.pth',  # noqa: E501
+        style='pytorch',
+        conv_cfg=dict(type='Conv3d'),
+        norm_cfg=None,
+        act_cfg=dict(type='ReLU'),
+        dropout_ratio=0.5,
+        init_std=0.005),
+    cls_head=dict(
+        type='I3DHead',
+        num_classes=101,
+        in_channels=4096,
+        spatial_type=None,
+        dropout_ratio=0.5,
+        init_std=0.01))
+# model training and testing settings
+train_cfg = None
+test_cfg = dict(average_clips='score')
+# dataset settings
+dataset_type = 'RawframeDataset'
+data_root = 'data/ucf101/rawframes'
+data_root_val = 'data/ucf101/rawframes'
+split = 1  # official train/test splits. valid numbers: 1, 2, 3
+ann_file_train = f'data/ucf101/ucf101_train_split_{split}_rawframes.txt'
+ann_file_val = f'data/ucf101/ucf101_val_split_{split}_rawframes.txt'
+ann_file_test = f'data/ucf101/ucf101_val_split_{split}_rawframes.txt'
+img_norm_cfg = dict(mean=[0, 0, 0], std=[1, 1, 1], to_bgr=False)
+# download c3d_mean_file from 'https://download.openmmlab.com/mmaction/'
```

Comment:
curios why they do this. i saw the file has the shape `(3, 16, 128, 171)`, isn't this too much? will this tuned means applicable to other datasets?

so the resolution is also fixed by this? do you have resize if the input resolution are larger?

Reply:
Thank you for your code review. I'm also confused about it. The file is not applicable to other datasets and it is the volume mean of UCF-101.

The relevant explanation is in the C3D doc "https://docs.google.com/document/d/1-QqZ3JHd76JfimY4QKqOojcEaf5g3JS0lNh-FHTxLag/"
![image](https://user-images.githubusercontent.com/35596075/96668336-89e90b80-138d-11eb-85ba-4cc9435b5373.png)

However, I cannot find the definition of volume mean. From the compute file, the computing method seems traversing all `(16, 3, 128, 171)` clips in the dataset, and calculate the mean value of each position.

If I use the common image regularization method, the accuracy is 79.75 (40 epochs) and is smaller than 80.04 (30 epochs) using volume mean under the same parameter setting.

Reply:
So the resolution is also fixed. In the data pipeline, the image will be resized to (128, 171)

Reply:
I have removed the data augmentation method(subtract volume mean), since it is not so generic and the accuracy is not greatly improved.